app/src/main/java/org/fossasia/openevent/api/APIClient.java

```diff
@@ -36,10 +37,11 @@ public APIClient() {
 
         RestAdapter adapter = new RestAdapter.Builder()
                 .setConverter(new GsonConverter(gson))
-                .setEndpoint(BASE_URL)
-                .setLogLevel(RestAdapter.LogLevel.BASIC)
+                .setEndpoint(Urls.BASE_GET_URL)
+                .setLogLevel(RestAdapter.LogLevel.FULL)
                 .build();
         openEventAPI = adapter.create(OpenEventAPI.class);
+        Log.d("APICLIENT" , openEventAPI.toString());
```

Comment:
Yeah I did figure that out earlier but I didn't remove this log. I am changing it already